plugins/modules/k8s_log.py

```diff
@@ -159,6 +167,12 @@ def execute_module(module, k8s_ansible_mixin):
     if module.params.get('container'):
         kwargs['query_params'] = dict(container=module.params['container'])
 
+    if module.params.get('since_seconds'):
+        if not kwargs['query_params']:
+            kwargs['query_params'] = dict(sinceSeconds=module.params['since_seconds'])
+        else:
+            kwargs['query_params'].update(dict(sinceSeconds=module.params['since_seconds']))
```

Comment:
```suggestion
        kwargs.setdefault('query_params', {}).update({'sinceSeconds': module.params['since_seconds']})

```
This will raise a KeyError if the `container` param is not set.